docs/networking/connecting-the-arena-to-a-network.md

```diff
@@ -1,19 +1,19 @@
 ---
 id: connecting-the-arena-to-a-network 
-title: Connecting the Arena to a network
-sidebar_label: Connecting the Arena to a network
+title: Connecting the Diamond 9 or Arena to a network
+sidebar_label: Connecting the Diamond 9 or Arena to a network
 ---
 
-The Arena console is different to the other Titan consoles because it
-has a built-in network switch and an optical network connection
-(optionally 2 optical connections). **This section applies only to the
-Arena console.**
+The Diamond 9 and Arena consoles are different to the other Titan consoles because they
+have a built-in network switch and an optical network connection. 
+**This section applies only to the Diamond 9 and 
+Arena consoles.**
 
 Optical Connections
 -------------------
 
 The optical network connector is a Neutrik opticalCON Duo which uses
-multimode fibre. A single connector is fitted as standard, but a second
+multimode fibre. On the Arena a single connector is fitted as standard, but a second
```

Comment:
Might be worth double-checking that the D9 SFPs are multimode if you haven't already - I've pinged JB a message, just waiting to hear back.

Reply:
Have confirmed, is multimode. Ignore me!